### ISSUE TITLE
[1.8.8] iPhone 6 / iOS 9.3.1 - App Crash - Swapping stickers in create lomotif

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#347]
Frequency: Every time

Action:
1. Launch Lomotif
2. Tap on start
3. Select 4 - 5 videos from camera roll or few motifs from any motif category
4. Click on apply button from top right corner
5. Tap on Find Music tab from Find Music screen
6. Select any song from Find Music and Apply
7. Now, click on Stickers tab
8. Select any sticker from the available stickers
9. Change the stickers and select first sticker from the list
10. Swap the stickers continuously

Expected result: Stickers should be changed during swapping and should be displayed on the screen

Actual result: Application Crash

[Crash Logs](https://drive.google.com/drive/u/0/folders/0B6P2U77GNA4Uc0ZxOXJBdkpENmc)

**Comment by xiuie** [comment]
@hiteshbheda Does this still happen in 1.8.9?

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa I can reproduce this issue for build [1.8.9] on iPhone 6 / iOS 9.3.1

**Comment by xiuie** [comment]
I can't reproduce this on my iPhone 6 / 9.2.1

**Comment by hiteshbheda** [comment]
@xiuie Yes Xiu, still i can replicate it as still there is a problem with FIRST STICKER.

**Comment by hiteshbheda** [comment]
@xiuie Xiu, Please find attached video. It will help you to understand the issue better.

[Video](https://drive.google.com/drive/u/0/folders/0B6P2U77GNA4UaTl6dkJOU042Tlk)

**Comment by vysohanych** [comment]
I can reproduce it on build 1.8.9 on iPad 2 iOS 9.3.1

**Comment by Dipamit** [comment]
I am not able to reproduce this issue on iPhone 5/8.3 for build [1.8.10]

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa I can't replicate this issue for build [1.8.10] on iPhone 6 / iOS 9.3.1

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa I can't replicate this issue for build [1.8.12] on iPhone 6 / iOS 9.3.1

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa I can't reproduce this issue for build [1.8.14] on iPhone 6 / iOS 9.3.1

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa I can't reproduce this issue for build [1.8.24] on iPhone 6 / iOS 9.3.2

**Comment by hiteshbheda** [comment]
@xiuie I can't reproduce this issue for build [2.1.4] on iPhone 6s / iOS 10.1.1
